scripts/kms/endpoints/heartbeat.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+heartbeat() {
```

Comment:
Good news, they can! They just use curl so if you've set the endpoints and paths correctly it works! 

The endpoints which have multiple authentication options include an arg so that you can specify which to use (see `scripts/kms/endpoints/key.sh`)

Although as mentioned in my above comment, adding support for AKV based keys would be a great addition so I suggest we raise an issue to add that